app/models/opentofu_worker.rb

```diff
@@ -74,10 +74,32 @@ def unit_environment_variables
 
   def configure_service_worker_deployment(definition)
     super
-
+    # overwriting container port to be same as opentofu-runner service port i.e. in this case 6000
     definition[:spec][:template][:spec][:containers].first[:ports] = [{:containerPort => container_port}]
 
+    # ovewriting home directory to terraform home dir
+    env_var_array = definition[:spec][:template][:spec][:containers][0][:env]
+    var_name = "HOME"
+    updated_value = "/home/node"
+    env_var_array.each do |env_var_key_val|
+      if env_var_key_val[:name] == var_name
+        env_var_key_val[:value] = updated_value
+        break
+      end
+    end
+
+    # setting up runasuser
+    definition[:spec][:template][:spec][:containers][0][:securityContext][:runAsUser] = 1000690001
+    # these volume mounts are require by terraform runner to create the stack, mentioned it as {} so that it can be writable
+    definition[:spec][:template][:spec][:containers].first[:volumeMounts] << {:name => "terraform-bin-empty", :mountPath => "/home/node/terraform/bin"}
```

Comment:
Just cosmetic but can we standardize on either `[:containers][0]` or `[:containers].first` here?

I see we use both in core, but in the same method it'd be nice if we used the same one.